frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/secret.tsx

```diff
@@ -63,7 +63,11 @@ export const GetSecret: React.FC<GetSecretProps> = ({ obj }) => {
   const dl = secretValues.length
     ? secretValues.reduce((acc, datum) => {
         const { field, value } = datum;
-        acc.push(<dt key={`${field}-k`}>{field}</dt>);
+        acc.push(
+          <dt key={`${field}-k`} data-test="secret-data__dt">
```

Comment:
```suggestion
          <dt key={`${field}-k`} data-test="secret-data">
```
data-test doesn't need to adhere to BEM naming convention.